frontend/challedu-app/src/components/welcome.js

```diff
@@ -0,0 +1,26 @@
+import React, { Component } from "react";
+import { Button }  from 'reactstrap';
+import 'bootstrap/dist/css/bootstrap.min.css';
+import '../App.css';
+import logo from '../images/logo.svg';
+
+const buttonStyle = {maxWidth: 200, margin: '0 auto 10px'};
+
+
+
+export default class welcome extends Component {
+    render() {
+        return (
+            <div class="welcome">
+                <img id="logo" src={logo} alt="logo"/>
+                <div id="welcome_msg">
+                    <h3>Welcome to our platform</h3>
+                </div>
+                <div id="welcome_buttons" style={buttonStyle}>
```

Comment:
you can override the style of the button component using the props, no need for passing this inline style to the parent div

Reply:
there is no override, id and style have different task.